app/controllers/tree_controller.rb

```diff
@@ -56,7 +56,7 @@ def automate_find_hierarchy(fqname)
     begin
       # Collect all the db records based on the parsed fqname
       open_nodes = namespace.split('/').each_with_object([]) do |ns, items|
-        items << MiqAeNamespace.find_by!(:name => ns, :parent => items.last)
```

Comment:
If `items.last` is a namespace object, this is the correct way to specify a parent.

`:ancestry` is column that stores all path ids from the root to the parent for every node, in the format like `"100018/100019"`.

What are the steps to re-create the issue? Thanks.

Reply:
I create customization dialog, add dynamic fields, assign automation instance to fiield, save dialog
Next i tried to change assigned instance and got error.